tests/std/tests/P0355R7_calendars_and_time_zones_clocks/test.compile.pass.cpp

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <cassert>

```

Comment:
Ultra nitpick: This test doesn't use the `assert` macro, only the `static_assert` keyword.